spec/memory-safe-d.dd

```diff
@@ -57,19 +57,31 @@ $(H2 $(LNAME2 usage, Usage))
          passed to functions. Such pointers include: raw pointers, arrays, `this`, classes, `ref` parameters, delegate/lazy parameters,
          and aggregates containing a pointer.)
 
+         $(P The $(DDSUBLINK spec/function, return-ref-parameters, `return ref`) attributes on a parameter indicates:)
+
+         * A pointer derived from the parameter's address may be returned from the function
+           (e.g. returning the parameter by reference).
+         * A pointer derived from the parameter's address may be stored in the first parameter of
+           a function returning `void`, *iff* the first parameter is `ref`.
```

Comment:
Or the implicit `this` parameter. Perhaps instead of repeating the exact rules, say 'or returned by assignment to another parameter' with a link to the exact rules.

Reply:
Updated.